modules/web/pom.xml

```diff
@@ -136,6 +136,21 @@
             </plugin>
         </plugins>
     </build>
+    
+    <profiles>
+        <profile>
+            <id>platform-windows</id>
+            <activation>
+                <os>
+                    <family>windows</family>
+                </os>
+            </activation>
+            <properties>
+                <!-- Override the executable names for Windows -->
+                <npm.executable>npm.cmd</npm.executable>
+            </properties>
+        </profile>
+    </profiles>
```

Comment:
I think we need to add the default property as well : 

`	
I used this workaround to have a cross-platform Maven build : declare the npm executable name as a Maven variable, and use Maven filters to modify this executable name when running on Windows.

It can work the same for Grunt, Bower etc.

<properties>
    <npm.executable>npm</npm.executable>
</properties>`

http://stackoverflow.com/questions/22708255/cannot-run-program-npm-in-directory